redisinsight/ui/src/pages/rdi/pipeline-management/components/jobs-panel/Panel.tsx

```diff
@@ -111,7 +110,7 @@ const DryRunJobPanel = (props: Props) => {
         id: rdiInstanceId,
       },
     })
-    dispatch(rdiDryRunJob(rdiInstanceId, JSON.parse(input), yaml.load(job)))
+    dispatch(rdiDryRunJob(rdiInstanceId, JSON.parse(input), job))
```

Comment:
can we fail here with JSON.parse?

Reply:
Nope, we validate submit button. If JSON.parse failed - the button will be disabled